pkg/defaults/defaults.go

```diff
@@ -109,8 +104,6 @@ const (
 
 	DefaultGCPModel = "GCP"
 
-	DefaultEVEName = "node1"
```

Comment:
Wait, why are we removing all this @giggsoff ?

Reply:
We use them in tests/integration. These variables are not required now.